global_objects/customer_button.py

```diff
@@ -0,0 +1,109 @@
+from typing import Dict, List, Optional, Any
+
+from ..business_objects import general
+from ..session import session
+from ..models import CustomerButton
+from .. import enums
+from sqlalchemy.orm.attributes import flag_modified
+
+
+def get(id: str) -> CustomerButton:
+    return (
+        session.query(CustomerButton)
+        .filter(
+            CustomerButton.id == id,
+        )
+        .first()
+    )
+
+
+def get_all(filter_visible: Optional[bool] = None) -> List[CustomerButton]:
+
+    where_add = ""
+
+    if filter_visible is not None:
```

Comment:
maybe we should sanitize the filter visible, even if with the current implementation boolean is enforeced on the request level